server/src/test/java/com/broadcom/lsp/cobol/service/delegates/references/ElementOccurrencesTest.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package com.broadcom.lsp.cobol.service.delegates.references;
+
+import com.broadcom.lsp.cobol.service.CobolDocumentModel;
+import com.broadcom.lsp.cobol.service.delegates.validations.AnalysisResult;
+import org.eclipse.lsp4j.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```

Comment:
Missing javadoc

Reply:
I'll try to change style check to catching this #715